addons/bestja_detailed_reports/views/detailed_reports_summary.xml

```diff
@@ -65,6 +67,21 @@
                 ('detailed_report.state', '!=', 'draft'),
             ]</field>
             <field name="context">{
+                'search_default_responsible_project': active_id,
+                'search_default_accepted': True,
+            }</field>
+        </record>
+
+        <record model="ir.actions.act_window" id="action_detailed_reports_product_summary_lvl0">
```

Comment:
This pull request is a dual pull request to previous pull request on estimation reports.